sky/cli.py

```diff
@@ -1459,39 +1503,76 @@ def status(all: bool, refresh: bool, clusters: List[str]):  # pylint: disable=re
+                                      refresh=refresh)
+        nonreserved_cluster_records = []
+        reserved_clusters = []
+        for cluster_record in cluster_records:
+            cluster_name = cluster_record['name']
+            if cluster_name in backend_utils.SKY_RESERVED_CLUSTER_NAMES:
+                reserved_clusters.append(cluster_record)
+            else:
+                nonreserved_cluster_records.append(cluster_record)
+        local_clusters = onprem_utils.check_and_get_local_clusters(
+            suppress_error=True)
 
-    num_pending_autostop = 0
-    num_pending_autostop += status_utils.show_status_table(
-        nonreserved_cluster_records, all)
-    for cluster_group_name, cluster_record in reserved_clusters.items():
+        num_pending_autostop = 0
         num_pending_autostop += status_utils.show_status_table(
-            [cluster_record], all, reserved_group_name=cluster_group_name)
-    if num_pending_autostop > 0:
-        plural = ' has'
-        if num_pending_autostop > 1:
-            plural = 's have'
-        click.echo('\n'
-                   f'{num_pending_autostop} cluster{plural} '
-                   'auto{stop,down} scheduled. Refresh statuses with: '
-                   f'{colorama.Style.BRIGHT}sky status --refresh'
-                   f'{colorama.Style.RESET_ALL}')
-    status_utils.show_local_status_table(local_clusters)
+            nonreserved_cluster_records + reserved_clusters, all)
+        status_utils.show_local_status_table(local_clusters)
+
+        hints = []
+        if show_spot_queue:
+            click.echo(f'\n{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
+                       f'Managed spot jobs{colorama.Style.RESET_ALL}')
+            with backend_utils.safe_console_status(
+                    '[cyan]Checking spot jobs[/]'):
+                n_jobs, msg = spot_jobs_future.get()
+
+                try:
+                    pool.close()
+                    pool.join()
+                except SystemExit as e:
+                    # This is to avoid a "Exception ignored" problem caused by
+                    # ray worker setting the sigterm handler to sys.exit(15)
+                    # (see ray/_private/worker.py).
+                    if e.code != 15:
+                        raise
+
+            click.echo(msg + '\n')
+            if n_jobs is not None:
+                # spot controller is UP.
+                job_info = ''
+                if n_jobs > 0:
+                    job_info = f'{n_jobs} spot jobs are in progress'
+                    if n_jobs > _SPOT_JOBS_IN_STATUS:
+                        job_info += (f' ({_SPOT_JOBS_IN_STATUS} latest ones '
+                                     'shown above)')
+                    job_info += '. '
+                hints.append(
+                    f'* {job_info}To see all jobs: {colorama.Style.BRIGHT}'
+                    f'sky spot queue{colorama.Style.RESET_ALL}')
```

Comment:
This may conflict with 
```
» sst
Clusters
NAME                          LAUNCHED     RESOURCES                            STATUS  AUTOSTOP  COMMAND                      
sky-spot-controller-8a3968f2  29 mins ago  1x GCP(n2-standard-4, disk_size=50)  UP      10m       sky spot launch -dy --cloud...

Managed spot jobs
No in progress jobs.

* To see all jobs: sky spot queue
* 1 cluster has auto{stop,down} scheduled. Refresh statuses with: sky status --refresh
```
Maybe in this case, we can skip the first hint, or show `* To see all jobs: sky spot queue`?

Reply:
Ahh, actually, I intended to show the `sky spot queue` so that the user can see all the spot jobs including those finished ones. Otherwise, they may be a bit confused about where the finished jobs go. Wdyt?

Reply:
Yes, not feeling strongly.